toolbox/tests/bulk_processing/test_uninvalidate_address_processor.py

```diff
@@ -0,0 +1,52 @@
+from unittest.mock import patch
+
+import pytest
+
+from toolbox.bulk_processing.bulk_processor import BulkProcessor
+from toolbox.bulk_processing.uninvalidate_address_processor import UnInvalidateAddressProcessor
+
+
+@pytest.mark.parametrize('case_id',
+                         [('test_case_id', 'TEST'),
+                          ('anything', 'BLAH'), ])
```

Comment:
```suggestion
@pytest.mark.parametrize('case_id', ['test_case_id', 'TEST', 'anything', 'BLAH'])
```
You're currently only passing one parameter (`case_id`) in but the test cases are tuples of two so the second value is just being ignored.